modular_citadel/code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -0,0 +1,17 @@
+/obj/structure/closet/secure_closet/warden/PopulateContents()
+	new /obj/item/device/radio/headset/headset_sec(src)
+	new /obj/item/clothing/suit/armor/vest/warden(src)
+	new /obj/item/clothing/head/warden(src)
+	new /obj/item/clothing/head/beret/sec/navywarden(src)
+	new /obj/item/clothing/suit/armor/vest/warden/alt(src)
+	new /obj/item/clothing/under/rank/warden/navyblue(src)
+	new /obj/item/clothing/glasses/hud/security/sunglasses(src)
+	new /obj/item/holosign_creator/security(src)
+	new /obj/item/clothing/mask/gas/sechailer(src)
+	new /obj/item/storage/box/zipties(src)
+	new /obj/item/storage/box/flashbangs(src)
+	new /obj/item/storage/belt/security/full(src)
+	new /obj/item/device/flashlight/seclite(src)
+	new /obj/item/clothing/gloves/krav_maga/sec(src)
+	new /obj/item/door_remote/head_of_security(src)
```

Comment:
You can replace all of this with ..()

Reply:
Would that not make the combat shotgun also spawn in? Unless that's the intention, of course.